include/miopen/config.h.in

```diff
@@ -73,6 +73,7 @@
 //     to BF16 results. This affects the main functionality of the library.
 #cmakedefine01 MIOPEN_USE_RNE_BFLOAT16
 
+#cmakedefine MIOPEN_DEFAULT_FIND_MODE "@MIOPEN_DEFAULT_FIND_MODE@"
```

Comment:
I that that if you remove double quotes, then you'll be able to use the macro where the value of `FindMode::Values::Default_` is defined. The value of macro would be an identifier (not literal C string). The changes in find_controls.cpp won't be necessary.